src/domain/mongoose_domain_core.erl

```diff
@@ -94,6 +95,9 @@ get_all_static() ->
 get_domains_by_host_type(HostType) when is_binary(HostType) ->
     heads(ets:match(?TABLE, {'$1', HostType, '_'})).
 
+get_all_domains() ->
+    heads(ets:match(?TABLE, {'$1', '$2', '_'})).
+
```

Comment:
Sorry but I just overlooked the simplest solution. Instead of this function we could just do:

```suggestion
domain_count() ->
    ets:info(?TABLE, size).
```